### PR TITLE
MDXv2: update e2e and benchmark

### PR DESCRIPTION
**Pull request by axe312ger**
This refactors the e2e tests and the benchmark suite to match the current state of the plugin

### CODE REVIEW

File: e2e-tests/mdx/src/pages/frontmatter-graphql-query.js [hunk 1]
```diff
@@ -8,23 +8,16 @@ export default function PageRunningGraphqlResolversOnJSFrontmatterTestInputs({
...
-          paragraphs
-          sentences
-          words
-        }
-        mdxAST
-      }
-    }
-  }
-`
+         {
```
Comment (LekoArts) [inline e2e-tests/mdx/src/pages/frontmatter-graphql-query.js:11]: Indentation is off here

File: e2e-tests/mdx/gatsby-node.js [hunk 2]
```diff
@@ -15,17 +15,4 @@ exports.onPostBuild = async ({ graphql }) => {
...
-  createTypes(`
-    type Mdx implements Node {
-      frontmatter: Frontmatter
-    }
-    type Frontmatter {
-      title: String
-    }
-  `)
-}
+}
```
Comment (LekoArts) [inline e2e-tests/mdx/gatsby-node.js:18]: Why did you remove that?

Comment (axe312ger) [reply e2e-tests/mdx/gatsby-node.js:18]: Should be not needed? The plugin now takes care of the schema?

File: e2e-tests/mdx/src/pages/frontmatter-graphql-query.js (hunk 1 above)
Comment (axe312ger) [reply e2e-tests/mdx/src/pages/frontmatter-graphql-query.js:11]: This is what prettier wanted 🤔

Comment (axe312ger) [reply e2e-tests/mdx/src/pages/frontmatter-graphql-query.js:11]: Nice, it formats it back and forth in my editor :D

Forced it now to be aligned left!

File: e2e-tests/mdx/gatsby-node.js (hunk 2 above)
Comment (LekoArts) [reply e2e-tests/mdx/gatsby-node.js:18]: ah, ok 👍

**Review by LekoArts — APPROVED** [review]
